### DISCUSSION TITLE
Templates for fediverse posts, or just remove attached images

### DISCUSSION THREAD

**Discussion by fivestone** [bookwyrm-social/bookwyrm#2758] [Ideas]
hi, is it possible to modify the template while posting quotes or reading notes to fediverse? (on user interface, or just code level for self-hosters)

and, may I remove the book cover image that attached in each reading note post? That seems annoying.

Thanks.

**Comment by jaschaurbach** [comment]
I would suggest to create options for every of your demands. 

Maybe someone is willing to write an issue and someone else to create a pull request for your demand.
